gluecodium/src/main/resources/templates/jni/utils/JniBaseImplementation.mustache

```diff
@@ -25,6 +25,12 @@
 
 #include <pthread.h>
 
+// Allow to change JNI_OnLoad function name via build system since it is global and the user
+// might want to have custom implementation
+#ifndef GLUECODIUM_JNI_ONLOAD
+# define GLUECODIUM_JNI_ONLOAD
```

Comment:
Why is this needed to be duplicated in the implementation file? Isn't it enough to have it in the header? If the header define is missing, this one will be empty and will produce uncompilable code regardless.